com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/AbstractFileService.java

```diff
@@ -240,6 +240,8 @@ public Integer execute(Connection connection) throws SQLException {
             throw new FileStorageException(e.getMessage(), e);
         }
     }
+    
+    public abstract void deleteSpaceLeftovers(final String space) throws FileStorageException;
```

Comment:
Isn't this the spaceId?

Reply:
Why not "deleteBySpace"? "deleteSpaceLeftovers" adds some context coming from the clean-up job's perspective, which should not be in the FileService.